movies-app/src/components/features/auth/LoginForm/LoginForm.tsx

```diff
@@ -34,6 +36,7 @@ export const LoginForm = () => {
                 duration: 5000,
                 isClosable: true,
               });
+

```

Comment:
Have you noticed that it takes a while for your app to redirect after logging in?
What you should do about it is to update the user SWR cache with the mutate function.

Reply:
I've now added mutate(swrKeys.myProfile).